CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+- BREAKING: clap 2.33.3 -> 3.0.4 ([#289]).
```

Comment:
Are there any breaking changes in how the operator is used now?

Reply:
Users need to replace their clap 2 and structopt dependencies with clap 3, and replace references to `structopt::StructOpt` with `clap::Parser`.